django/contrib/staticfiles/templatetags/staticfiles.py

```diff
@@ -1,16 +1,29 @@
+import warnings
+
 from django import template
-from django.contrib.staticfiles.storage import staticfiles_storage
 from django.templatetags.static import StaticNode
+from django.templatetags.static import static as _static
+from django.utils.deprecation import RemovedInDjango20Warning
 
 register = template.Library()
 
 
 def static(path):
-    return staticfiles_storage.url(path)
+    # Implementation has been moved to django.templatetags.static.
```

Comment:
Maybe it could be worth to introduce a new type of DeprecationWarning ? Something like `WillBeDeprecatedInFuture`, that would be visible only to Django developers (did not check if it was possible with the `warnings` module though)...